framework/include/base/FEProblemBase.h

```diff
@@ -1297,6 +1297,11 @@ class FEProblemBase : public SubProblem, public Restartable
 
   ExecuteMooseObjectWarehouse<MultiApp> & getMultiAppWarehouse() { return _multi_apps; }
 
+  /**
+   * Turn off notifications on the screen from transfers finishing execution.
+   */
+  void silenceParallelBarriers() { _silence_parallel_barriers = true; }
```

Comment:
Make this a real setter, i.e. it should take `bool state = true` as a parameter. And maybe even rename it to `setSilence...`